certified-connectors/InEight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3049 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "InEight",
+    "description": "The InEight Connector provides a mechanism to create new data in InEight or extract data from InEight. It can be used to integrate other systems with InEight.",
+    "contact": {
+      "name": "InEight Support",
+      "url": "https://ineight.com/help-support/",
+      "email": "support@ineight.com"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://ineight.com/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://ineight.com/privacy-statement/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Data;Productivity"
+    }
+  ],
+  "host": "api.ineight.com",
+  "basePath": "/integrations/v1",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "oauth2_auth": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://login.windows.net/common/oauth2/authorize",
+      "tokenUrl": "https://login.microsoftonline.com/common/oauth2/token",
+      "scopes": {}
+    }
+  },
+  "security": [
+    {
+      "oauth2_auth": []
+    }
+  ],
+  "paths": {
+    "/api/DailyPlan": {
+      "get": {
+        "description": "The Daily Plan integration allows customers to provide time keeping information for employees and equipment as recorded in InEight Plan. This integration includes summary information about the daily plan, from employee hours to cost items, equipment hours to cost items, payroll indicators, and notes. <br/>The Daily Plan integration can also be used to track and provide plan status in systems outside of InEight cloud platform. When a daily plan is used for this purpose, an optional response message can be used to trigger a status change in the daily plan to indicate the change has been successfully received by the external system.",
+        "operationId": "DailyPlan_Get",
+        "summary": "DailyPlan_Get",
```

Comment:
@InEightDevOps Summary values are visible to the public and these values must be free of grammatical and spelling errors.

Docs - [https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)